### PR TITLE
newsletter august 24

### PR DESCRIPTION
**Pull request by Romain-Thomas-Shef**
Sorry I am a week late for this.

### CODE REVIEW

File: _newsletters/2024-08-13-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
+---
+layout: post
+title: August 2024 Newsletter
+editor: Romain Thomas
+slug: 2024-08-13-newsletter
+date: 2024-07-13T17:00:00Z+0100
```
Comment (ptheywood) [inline _newsletters/2024-08-13-newsletter.md]:
```suggestion
date: 2024-08-13T17:00:00Z+0100
```

File: _newsletters/2024-08-13-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+
+
+- [CodeRefinery workshop](https://coderefinery.github.io/2024-09-10-workshop/)
+  - **2024/09/10-12 and 17-19**. Free workshop on git, testing, documentation, and more in a workshop - for researchers and scientists.
+
+- [Sustainable HPC State of the Practice](https://sites.google.com/lbl.gov/sustainable-hpc-sop-workshop24/home)
+ - **2024/09/24** The demand for ever more-capable high performance computing (HPC) is driving significant changes across the design and manufacturing space, as manufacturers turn to heterogeneous systems that integrate increasing die-count, from multi-core CPUs to accelerators, traditional memory architectures to high-bandwidth memory, modern interconnects, and massive storage servers. These designs require substantially higher energy, with commensurate methods for managing heat, in increasingly dense packages. To effectively manage and operate these systems, HPC and Data Center (DC) practitioners must balance and coordinate constraints across many domains: environment, utilities, data center, HPC system hardware and software, and end-user applications.
```
Comment (ptheywood) [inline _newsletters/2024-08-13-newsletter.md]:
```suggestion
  - **2024/09/24** The demand for ever more-capable high performance computing (HPC) is driving significant changes across the design and manufacturing space, as manufacturers turn to heterogeneous systems that integrate increasing die-count, from multi-core CPUs to accelerators, traditional memory architectures to high-bandwidth memory, modern interconnects, and massive storage servers. These designs require substantially higher energy, with commensurate methods for managing heat, in increasingly dense packages. To effectively manage and operate these systems, HPC and Data Center (DC) practitioners must balance and coordinate constraints across many domains: environment, utilities, data center, HPC system hardware and software, and end-user applications.
```

Needs 2 spaces not 1 for this to be a child bullet

File: _newsletters/2024-08-13-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+<!--If a course is "sold out" please join the wait list by signing up - we regularly email people to encourage those -->
+<!--that can no longer attend to cancel. Those on the wait list get early notification when the courses are run -->
+<!--again.-->
+
+<!-- ### Opportunities -->
+
+<!-- - [Description](https://www.elsewhere.ac.uk/): -->
+<!--   - More Details (optional) -->
+
+#### Jobs
```
Comment (ptheywood) [inline _newsletters/2024-08-13-newsletter.md]: This is nested below Training currently. 

Either need to uncomment `### Opportunities` above, or make Jobs a h3. 

```suggestion
### Jobs
```

**Review by ptheywood — CHANGES_REQUESTED** [review]

File: _newsletters/2024-08-13-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+## The University of Sheffield Research Software Engineering Community Newsletter August 2024
+
+Welcome to the August 2024 newsletter for the research software community at The University of Sheffield, featuring
+news, opportunities, events and training for you.
+
+
+
+### News
+
+Thank you to everyone who applied for the [2024 RSE collaboration call for proposals](https://rse.shef.ac.uk/collaboration/RSEtime_call2024/). This call is now closed. We had 26 applications! We have reviewed all proposals and sent notifications to all applicants.
```
Comment (ns-rse) [inline _newsletters/2024-08-13-newsletter.md]: Would we be happy announcing the projects that were awarded support? If so...

```suggestion
Thank you to everyone who applied for the [2024 RSE collaboration call for proposals](https://rse.shef.ac.uk/collaboration/RSEtime_call2024/). This call is now closed. 

We received 26(!) applications from across the University. We have reviewed all proposals and sent notifications to applicants. The successful projects that will receive six months 0.5 FTE RSE support are...

- PolyChron: A tool for archaeologists for the management of absolute and relative dating evidence - Bryony Moody (School of Mathematics)
- HYBIRD: a Particle-Fluid Engineering Simulation Tool (HYPFEST) - Alessandro Leonardi (Civil Engineering)
- Tools to analyse splice-aware SLAMseq - Ian Sudbery (School of Biosciences)
```

**Review by ns-rse — APPROVED** [review]
Looks good, just one suggestion to expand on the call for proposals  if we wanted to, not essential though.

**Review by ptheywood — APPROVED** [review]

**Pull request merged**